powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/ActivationServiceBehavior.java

```diff
@@ -1319,6 +1324,7 @@ public CreateActivationResponse createActivation(CreateActivationRequest request
             initRequest.setTimestampActivationExpire(activationExpireTimestamp);
             initRequest.setActivationOtp(activationOtp);
             initRequest.setActivationOtpValidation(activationOtpValidation);
+            initRequest.setCommitPhase(com.wultra.security.powerauth.client.model.enumeration.CommitPhase.ON_COMMIT);
```

Comment:
Indeed, this can happen and conflicting values are checked here:
https://github.com/wultra/powerauth-server/pull/1671/files#diff-a4ce67a9fd8bc9dfacab0302fa5f1cb1a44ee59e9ceafb746ef9e8df877d7492R987